packages/node_modules/@webex/test-users/test/integration/spec/index.js

```diff
@@ -142,20 +149,42 @@ nodeOnly(describe)('test-users', () => {
     it('retrieves credentials with a useable refresh token', () => createTestUser()
       .then(prune)
       .then(loginTestUser)
-      .then((token) => {
+      .then(async (token) => {
         assert.hasAccessToken({token});
         assert.hasRefreshToken({token});
 
-        return assert.isFulfilled(refresh({token}));
+        const res = await refresh({token});
+
+        assert.equal(res.statusCode, 200);
       }));
   });
 
   describe('removeTestUser()', () => {
     it('removes the specified test user', () => createTestUser()
-      .then((u) => assert.isFulfilled(removeTestUser(u))));
```

Comment:
I can't find any documentation on `assert.isFulfilled(..)`. The only mentioned I found on the _chai_ webpage is

https://www.chaijs.com/plugins/chai-as-promised/

But it does not do what either I, or these tests, expect. Namely to wait until the promise completes and assert it resolved.

Instead I found this test reached the `loginTestUser(..)` request POST step and then passed. It did not wait for a response but assumed the test succeeded. This meant the tests were checking nothing.